app-misc/neofetch/neofetch-3.3.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A CLI system information tool written in BASH that supports displaying images"
+DESCRIPTION="Neofetch displays information about your system next to an image, your OS logo, or any ASCII file of your choice. The main purpose of Neofetch is to be used in screenshots to show other users what OS/Distro you're running, what Theme/Icons you're using etc"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+COPYRIGHT="2016-2017 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dylanaraps/$portName/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="4808e76bd81da3602cb5be7e01dfed8223b1109e2792755dd0d54126014ee696"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	neofetch
+	cmd:neofetch
+	"
+REQUIRES="
+	haiku
+	cmd:bash
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:make
+	"
```

Comment:
Oups, I just noticed it would be better to split this into BUILD_REQUIRES and BUILD_PREREQUIRES:
```diff
 BUILD_REQUIRES="
 	haiku_devel
+	"
+BUILD_PREREQUIRES="
 	cmd:make
 	"
```

Reply:
Will fix, but i think BUILD_"PRE"REQUIRES should be above BUILD_REQUIRES

Reply:
Well, it's the opposite ;-)
[HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)